polaris/ocean/tasks/geostrophic/geostrophic.cfg

```diff
@@ -0,0 +1,125 @@
+# Options related to the vertical grid
+[vertical_grid]
+
+# the type of vertical grid
+grid_type = uniform
+
+# Number of vertical levels
+vert_levels = 1
+
+# Depth of the bottom of the ocean
+bottom_depth = 2000.0
+
+# The type of vertical coordinate (e.g. z-level, z-star)
+coord_type = z-star
+
+# Whether to use "partial" or "full", or "None" to not alter the topography
+partial_cell_type = None
+
+# The minimum fraction of a layer for partial cells
+min_pc_fraction = 0.1
```

Comment:
Are partial bottom cells something we want to expose in the config file for this case?

Reply:
You're right, we could take those out since it wouldn't make sense to use them in this configuration.  I copied them from cosine bell but the same applies there.